modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2762,6 +2762,40 @@ def explode(self, axis: Union[int, Axis], func: Callable) -> "PandasDataframe":
             partitions, new_index, new_columns, row_lengths, column_widths
         )
 
+    def combine(self) -> "PandasDataframe":
+        """
+        Create a single partition PandasDataframe from the partitions of the current dataframe.
+
+        Returns
+        -------
+        PandasDataframe
+            A single partition PandasDataframe.
+        """
+        partitions = self._partition_mgr_cls.to_single_partition(self._partitions)
```

Comment:
```suggestion
        partitions = self._partition_mgr_cls.combine(self._partitions)
```
Let's align the naming then.